json_serializable/test/default_value/default_value.g.dart

```diff
@@ -92,9 +94,18 @@ Map<String, dynamic> _$DefaultValueToJson(DefaultValue instance) =>
           instance.valueFromDefaultValueNamedConstructor,
     };
 
+// ignore: unused_element
 const _$GreekEnumMap = {
   Greek.alpha: 'alpha',
   Greek.beta: 'beta',
   Greek.gamma: 'gamma',
   Greek.delta: 'delta',
 };
+
+// ignore: unused_element
+const _$GreekEnumDecodeMap = {
```

Comment:
So we're changing the generated code and creating a NEW map, even if the dev doesn't use the new feature?